README.md

```diff
@@ -201,7 +201,7 @@ Restful.js exposes similar methods on collections, members and entities. The nam
 * `get ( id [, params [, headers ]] )`: Get a member in a collection. Returns a promise with an entity.
 * `post ( data [, headers ] )`: Create a member in a collection. Returns a promise with the response.
 * `put ( id, data [, headers ] )`: Update a member in a collection. Returns a promise with the response.
-* `delete ( id [, headers ] )`: Delete a member in a collection. Returns a promise with the response.
+* `delete ( id [, headers, data ] )`: Delete a member in a collection. Returns a promise with the response.
```

Comment:
You should update the documentation according to your last commit:
`delete ( id [, data, headers ] )`